packages/fluentui/react-northstar-prototypes/src/prototypes/MenuList/MenuButton.tsx

```diff
@@ -3,46 +3,84 @@ import { MenuList } from './MenuList';
 import { MenuItem } from './MenuItem';
 import { Menu } from './Menu';
 import { MenuTrigger } from './MenuTrigger';
+import { MenuDivider } from './MenuDivider';
+import { MenuSectionHeader } from './MenuSectionHeader';
+import { MenuGroup } from './MenuGroup';
+import { MenuItemCheckbox } from './MenuItemCheckbox';
+import { MenuItemRadio } from './MenuItemRadio';
 
 export function MenuButton() {
+  const [checkedValues, setCheckedValues] = React.useState<Record<string, number[]>>({});
+  const onCheckedValuesChange = (checkedValues: Record<string, number[]>) => setCheckedValues(checkedValues);
```

Comment:
I was kinda confused in the beginning by the fact that there's 2 `checkedValues` (MenuButton and MenuList). But then I figure out this one here has just the simple purpose of displaying children's `checkedValues`. Maybe this one here could be named differently? But this is not an API, so not so important, just how I would like it :)

Reply:
It's also to illustrate how a consumer would use the `Menu` component to manage their state

the `checkedValues` in `MenuList` is purely for that autocontrolled api pattern